apps/preview/next/pages/showcases/MegaMenu/OpenOnClick.tsx

```diff
@@ -0,0 +1,263 @@
+import { CSSTransition } from 'react-transition-group';
+import classNames from 'classnames';
+import brandLogo from '@assets/vsf_logo_white.svg';
+import brandLogoSign from '@assets/vsf_logo_sign_white.svg';
+import sneakers from '@assets/sneakers-2.png';
+
+export default function OpenOnClick() {
+  const [open, setOpen] = useState(false);
+  const nodeRef = useRef(null);
+  const actionItems = [
+    {
+      icon: <SfIconShoppingCart />,
+      ariaLabel: 'Cart',
+    },
+    {
+      icon: <SfIconFavorite />,
+      ariaLabel: 'Wishlist',
+    },
+    {
+      label: 'Log in',
+      icon: <SfIconPerson />,
+      ariaLabel: 'Log in',
+    },
+  ];
+
+  const bannerDetails = {
+    image: sneakers,
+    title: 'Hot Deals. No sweat.',
+  };
+
+  const categoriesContent = [
+    {
+      heading: 'Activities',
+      items: [
+        {
+          title: 'Fitness',
+          link: '/',
+        },
+        {
+          title: 'Pilates',
+          link: '/',
+        },
+        {
+          title: 'Training',
+          link: '/',
+        },
+        {
+          title: 'Cardio workout',
+          link: '/',
+        },
+        {
+          title: 'Yoga',
+          link: '/',
+        },
+        {
+          title: 'All activities',
+          link: '/',
+        },
+      ],
+    },
+    {
+      heading: 'Categories',
+      items: [
+        {
+          title: 'Clothing',
+          link: '/',
+        },
+        {
+          title: 'Shoes',
+          link: '/',
+        },
+        {
+          title: 'Accessories',
+          link: '/',
+        },
+        {
+          title: 'Wearables',
+          link: '/',
+        },
+        {
+          title: 'Food & Drinks',
+          link: '/',
+        },
+        {
+          title: 'All categories',
+          link: '/',
+        },
+      ],
+    },
+    {
+      heading: 'Womens Deals',
+      items: [
+        {
+          title: 'Fitness Gear',
+          link: '/',
+        },
+        {
+          title: 'Outlet',
+          link: '/',
+        },
+        {
+          title: 'All Womens Deals',
+          link: '/',
+        },
+      ],
+    },
+  ];
+
+  return (
+    <div className="w-full h-full">
+      {open && <div className="fixed inset-0 bg-neutral-500 bg-opacity-50 transition-opacity" />}
+      <header className="flex justify-center w-full border-0 bg-primary-700 border-neutral-200 h-14 md:relative md:h-20 md:z-10">
+        <div className="flex items-center flex-nowrap justify-start h-full max-w-[1536px] w-full px-4 md:px-10">
+          <a href="/" aria-label="SF Homepage" className="inline-block text-white mr-2 lg:mr-10">
+            <img src={brandLogo.src} alt="Sf Logo" className="hidden lg:block mr-10 w-[12.5rem] h-[1.75rem]" />
+            <img src={brandLogoSign.src} alt="Sf Logo" className="hidden w-10 h-10 mr-4 md:block lg:hidden" />
+            <img src={brandLogoSign.src} alt="Sf Logo" className="block w-8 h-8 mr-4 min-w-8 md:hidden" />
```

Comment:
If you're just trying to make different images for different screen sizes, then you could use `<picture>` element with different [`<source>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/source) elements with proper `media` attributes.

Reply:
Or one image tag with `srcset` https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#resolution_switching_different_sizes

Reply:
@Szymon-dziewonski @dkacper same as below - entire navbar element were copied from what we have, so we should change it also for navbar blocks -> I can do it as part of this PR or should we have a seperate task for it?

Reply:
I'd change it within this PR.